### PR TITLE
Update i2cConfig to accept an object or number

### PR DESCRIPTION
**Pull request by rwaldron**
I tried writing tests and building the source with babel, but gulp just failed every time so I gave up. 

Here's the error I hit: 

```
$ gulp
[13:14:44] Using gulpfile ~/clonez/raspi-io/gulpfile.js
[13:14:44] Starting 'clean'...
[13:14:44] Starting 'lint'...
events.js:85
      throw er; // Unhandled 'error' event
            ^
Error: spawn eslint ENOENT
    at exports._errnoException (util.js:746:11)
    at Process.ChildProcess._handle.onexit (child_process.js:1053:32)
    at child_process.js:1144:20
    at process._tickCallback (node.js:355:11)
    at Function.Module.runMain (module.js:503:11)
    at startup (node.js:129:16)
    at node.js:814:3
```

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -383,7 +383,17 @@ class Raspi extends EventEmitter {
     }
   }
 
-  i2cConfig(delay) {
+  i2cConfig(options) {
+    var delay;
```
Comment (nebrius) [inline index.js]: Can you update this to use ```let``` instead? eslint will complain otherwise :)

**Comment by nebrius** [comment]
That error sounds like eslint isn't installed (```npm install -g eslint```). One small style nit, otherwise looks good

**Comment by nebrius** [comment]
I'll make sure this doesn't cause any bugs/compile issues on my end. I'd like to get this released before taking on #31, which will necessitate a major version bump.

**Comment by nebrius** [comment]
This fix is now live w/ raspi-io 3.4.0. Thanks for the PR Rick!

File: index.js (hunk 1 above)
Comment (rwaldron) [inline index.js]: Sure can, sorry about the old-timey slip ;)

Comment (nebrius) [inline index.js]: No worries, we're all busy :) I went ahead and updated it on my end, so we're all good

**Comment by rwaldron** [comment]
Sorry about the var/let thingy, I'll pay better attention next time :)

**Pull request merged**